Resources/Prototypes/_Nuclear14/Entities/Clothing/Belt/belts.yml

```diff
@@ -90,18 +90,12 @@
   parent: N14ClothingBeltStorageBase
   id: ClothingBeltNCRBandolier
   name: bandolier
-  description: A bandolier commonly used to carry single shells or cartridges.
+  description: A bandolier commonly used to carry single shells or cartridges, modified by the NCR for more capacity.
   components:
   - type: Sprite
     sprite: _Nuclear14/Clothing/Belt/bandolier.rsi
   - type: Clothing
     sprite: _Nuclear14/Clothing/Belt/bandolier.rsi
-  - type: Storage
```

Comment:
Was it intended to remove the storage whitelist and has this been tested to make sure it still holds ammo / doesn't hold nasties?

Reply:
> Was it intended to remove the storage whitelist and has this been tested to make sure it still holds ammo / doesn't hold nasties?

So I'm not the PR creator, but I did test this out real quick. It can hold every ammo type I tested and by virtue of being a pretty small storage space cannot hold anything major like a full rifle. Should be fine.